tests/integration/test_question_pages.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from tests.factories import QuestionFactory
+from tests import factories
+
+
+@pytest.mark.django_db
+def test_get_question_page(client):
+    question = QuestionFactory(text="How should funding be allocated?", slug="how-should-funding-be-allocated")
+
+    response = client.get(f"/questions/{question.slug}")
+    assert "How should funding be allocated?" in str(response.content)
+
+
+def generate_question_with_themes():
+    consultation = factories.ConsultationFactory(slug="c-slug")
+    consultation_response = factories.ConsultationResponseFactory()
+    section = factories.SectionFactory(consultation=consultation, slug="s-slug")
+    question = factories.QuestionFactory(
+        section=section,
+        slug="question-slug",
+        text="Is this an interesting question?",
+        multiple_choice_options=[],
```

Comment:
I think this is blank by default?

Reply:
By default it gets whatever was in the YAML file, but we're doing this differently now anyway.